config/suse/waagent.conf

```diff
@@ -70,11 +70,26 @@ OS.OpensslPath=None
 # Detect Scvmm environment, default is n
 # DetectScvmmEnv=n
 
+#
+# Lib.Dir=/var/lib/waagent
+
+#
+# DVD.MountPoint=/mnt/cdrom/secure
+
+#
+# Pid.File=/var/run/waagent.pid
+
+#
+# Extension.LogDir=/var/log/azure
+
+#
+# Home.Dir=/home
+
 # Enable RDMA management and set up, should only be used in HPC images
 # OS.EnableRDMA=y
 
 # Enable or disable goal state processing auto-update, default is enabled
-# AutoUpdate.Enabled=y
+AutoUpdate.Enabled=n
```

Comment:
@bear454 We need to leave `AutoUpdate.Enabled` set to true (aka "y"). It is best for Azure customers.